application/teams/__init__.py

```diff
@@ -1,8 +1,10 @@
 """Provides Team functionality and routes"""
 from flask import Blueprint
 
-from .teams_view import TeamView, TeamsView
+from .teams_view import TeamView, TeamsView, MembersView, AllTeamsView
 
 TEAMS = Blueprint('teams', __name__)
 TEAMS.add_url_rule('', view_func=TeamsView.as_view('teamsview'))
 TEAMS.add_url_rule('/<id>', view_func=TeamView.as_view('teamview'))
+TEAMS.add_url_rule('/<id>/members', view_func=MembersView.as_view('membersview'))
+TEAMS.add_url_rule('/all', view_func=AllTeamsView.as_view('allteamsview'))
```

Comment:
There should not be a public route to request all teams. Teams are specific to each user and at the current moment not publicly accessible, and `/all` is not RESTful (https://www.restapitutorial.com/lessons/restfulresourcenaming.html). See further below for context on routing.